src/components/SortBy/SortBy.jsx

```diff
@@ -14,24 +14,24 @@ class SortBy extends Component {
     sortBy: PropTypes.string,
   };
 
-  handleSort(e) {
-    e.preventDefault();
-    return cb => cb;
+  handleSort(type) {
+    const { dispatch } = this.props;
+    return () => dispatch(sort(type));
   }
 
   render() {
-    const { dispatch, sortBy } = this.props;
+    const { sortBy } = this.props;
     return (
       <div className={styles.filters}>
         <p className={styles.title}>Sort by: </p>
         <ul className={styles.filtersList}>
           <li className={cx({ active: sortBy === constants.SORT_BY_RELEASE })}>
-            <button className={styles.button} onClick={e => this.handleSort(e)(dispatch(sort(constants.SORT_BY_RELEASE)))}>
+            <button className={styles.button} onClick={() => this.handleSort(constants.SORT_BY_RELEASE)()}>
```

Comment:
you can just pass result of the `handleSort()`: `onClick={ this.handleSort(constants.SORT_BY_RELEASE) }`